jupyter_releaser/lib.py

```diff
@@ -413,12 +420,15 @@ def publish_assets(
         name = Path(path).name
         suffix = Path(path).suffix
         if suffix in [".gz", ".whl"]:
-            env = os.environ.copy()
-            env["TWINE_PASSWORD"] = twine_token
-            # NOTE: Do not print the env since a twine token extracted from
-            # a PYPI_TOKEN_MAP will not be sanitized in output
-            util.retry(f"{twine_cmd} {name}", cwd=dist_dir, env=env)
-            found = True
+            if name.startswith(
+                python_package_name
+            ):  # FIXME: not enough to know it's the right package
```

Comment:
The distribution file name is something like `my_package-0.1.2.tar.gz` or `my_package-0.1.2-py3-none-any.whl`, and we could imagine a package name being `my_package` and another one being `my_package_2`, and both would match this condition, so this is not great. Maybe we could include the version spec in the condition, but at first sight tests were failing when doing so.